src/main/java/org/opentripplanner/routing/algorithm/filterchain/deletionflagger/RemoveBikerentalWithMostlyWalkingFilter.java

```diff
@@ -25,7 +25,7 @@ public String name() {
   }
 
   @Override
-  public Predicate<Itinerary> predicate() {
+  public Predicate<Itinerary> flagForRemoval() {
```

Comment:
Is `flagForRemoval` a good name for a method that returns a `Predicate`?

Reply:
I am not aware of any best practice for naming functions, and predicates in particular. But the name should reflect what it does. So, the function "check if an itinerary should be flagged for removal". Names like `doFlagForRemoval` and `flagForRemoval` works for me, any other suggestions?

Reply:
What I'm trying to say that "flagForRemoval" is a definitive action but what we are asking is "shouldFlagForRemoval", isn't it?

Reply:
Thanks for the clarification. I think you you are right - my english is not good enough. I will change it.

Reply:
I used `shouldBeFlaggedForRemoval` - agree?